site/docs/components/landing/CheckoutDemo.tsx

```diff
@@ -17,7 +17,7 @@ export const checkoutDemoCode = `
       </Checkout>
     )
   }
-`;
+  `;
```

Comment:
without the indent here, the code snippet can be cut-off by the preview container. with the indents, the scroll will always take the reader to the bottom of the string.